misc/metrics/src/protocol_stack.rs

```diff
@@ -0,0 +1,86 @@
+use libp2p_core::multiaddr::{Multiaddr, Protocol};
+use prometheus_client::encoding::text::Encode;
+
+#[derive(Encode, Hash, Clone, Eq, PartialEq)]
+pub struct Label {
+    address_stack: String,
+}
+impl Label {
+    pub fn for_multi_address(ma: &Multiaddr) -> Self {
```

Comment:
```suggestion
    pub fn from_multiaddress(ma: &Multiaddr) -> Self {
```

I don't think using `for_` is a convention. Am I missing something?

Reply:
No, you're right that that is more idiomatic for rust and so it makes more sense. I'll be implementing From as mentioned below.

I think I landed on "for" without really thinking about it just because if I were describing the result/purpose of this method in English prose I would've likely used "for", but I'm not writing English prose here - I'm writing rust.